csrc/ops/composite.cpp

```diff
@@ -471,16 +471,20 @@ SdpfaFwdResult sdpfa_fwd(
 
   // Create a new Tensorview for cum_seq_q, cum_seq_k of shape (N + 1)
   auto newForCumulativeSeq = [&]() -> TensorView* {
-    IterDomain* batch_id = ops::newOutputIterDomain({query_domain.front()});
-    batch_id = IterDomain::resize(
+    IterDomain* batch_id = ops::newOutputIterDomain({query_domain.front(), key_domain.front(), value_domain.front()});
```

Comment:
Why is this change?

`key_domain.front(), value_domain.front()}`

Reply:
`ops::newOutputIterDomain` can use a series of iterdomains to construct a new iterdomain. If any of these are non-symbolic, the extents are set using that iterdomain. This is useful in resolving broadcast iterdomains.
In practice, this maybe unlikely so using just `query_domain.front()` is sufficient too.

This is similar to the construction of attention output and log_sumexp in L446-447, L461-462 which use the iterdomains corresponding to `N` from all inputs (q/k/v).

This was missing for `cum_seq_q/k`, so updating now.